src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1324,7 +1321,24 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		}
 		const Addon = await this._terminalInstanceService.getXtermWebglConstructor();
 		this._webglAddon = new Addon();
-		this._xterm.loadAddon(this._webglAddon);
+		try {
+			this._xterm.loadAddon(this._webglAddon);
+			this._storageService.store(SUGGESTED_RENDERER_TYPE, 'auto', StorageScope.GLOBAL, StorageTarget.MACHINE);
+		} catch {
+			const neverMeasureRenderTime = this._storageService.getBoolean(NEVER_MEASURE_RENDER_TIME_STORAGE_KEY, StorageScope.GLOBAL, false);
```

Comment:
When we fallback let's add a log service warn, otherwise it will be hard to tell what auto has selected or when you set webgl explicitly (I guess falling back is fine then)